<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Planejamentos</class>
 <widget class="QDialog" name="Planejamentos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cb_CatExists">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Categorias Existentes:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_newCat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicione o Nome da Categoria</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_addCat">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar Categoria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nameCategory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_editCat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar Nome
Categoria</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_deleteCat">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Deletar
Categoria</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
